src/integrationTests/createJoinCommunity.test.ts

```diff
@@ -0,0 +1,76 @@
+import { Crypto } from '@peculiar/webcrypto';
+import { assertReceivedCertificates } from './assertions';
+import {
+  createCommunity,
+  joinCommunity,
+  getCommunityOwnerData,
+} from './appActions';
+import { createApp } from './utils';
+import { AsyncReturnType } from '../utils/types/AsyncReturnType.interface';
+
+jest.setTimeout(600_000);
+const crypto = new Crypto();
+
+global.crypto = crypto;
+
+describe.skip('owner creates community', () => {
+  let owner: AsyncReturnType<typeof createApp>;
+
+  beforeAll(async () => {
+    owner = await createApp();
+  });
+
+  afterAll(async () => {
+    await owner.manager.closeAllServices();
+  });
+
+  test('Owner creates community', async () => {
+    await createCommunity({ userName: 'Owner', store: owner.store });
+  });
+});
+
+describe.skip('owner creates community and two users join', () => {
+  let owner: AsyncReturnType<typeof createApp>;
+  let userOne: AsyncReturnType<typeof createApp>;
+  let userTwo: AsyncReturnType<typeof createApp>;
+
+  beforeAll(async () => {
+    owner = await createApp();
+    userOne = await createApp();
+    userTwo = await createApp();
+  });
+
+  afterAll(async () => {
+    await owner.manager.closeAllServices();
+    await userOne.manager.closeAllServices();
+    await userTwo.manager.closeAllServices();
+  });
+
+  test('Owner creates community', async () => {
+    await createCommunity({ userName: 'Owner', store: owner.store });
+  });
+
+  test('Two users join community', async () => {
+    const ownerData = getCommunityOwnerData(owner.store);
+
+    await joinCommunity({
+      ...ownerData,
```

Comment:
It should be renamed to just data, it contains expectedPeersCount,  registrarAddress, optional registrar port, desired username, and also some owner data, because we check if we get the correct owner CA. We check all those things in joinCommunity, this joinCommunity is a wrapper around original joinCommunity and contains the whole process of Joining community, createing certificate etc, and has a lot of assertions inside itself.